content/creator/sdk7/2d-ui/ui_background.md

```diff
@@ -81,6 +83,48 @@ ReactEcsRenderer.setUiRenderer(() => (
 ))
 ```
 
+## Borders
+
+A few properties are used to set a border around a UI entity. These properties exist on the `uiTransform` component. They each allow you to set either a single value for all sides of the border, or different values for each side.
+
+- `borderColor`: The color to use on the entity, as a [Color4]({{< ref "/content/creator/sdk7/3d-essentials/color-types.md">}}) value.
+- `borderWidth`: The width of the border, as a number in pixels. It also supports values in percentages, for example `borderWidth: '2%'` will set the border width to 2% of the entity's width.
+- `borderRadius`: Use this property to give the corners of the entity a rounded border. It sets the radius of the corners in pixels. It also supports values in percentages, for example `borderRadius: '19%'` will set the border radius to 10% of the entity's width.
```

Comment:
```suggestion
- `borderRadius`: Use this property to give the corners of the entity a rounded border. It sets the radius of the corners in pixels.
```

i think we cant do this with unity right now, it only accepts points, and for a % we need to do an extra math that we currently dont support.
I have to remove the option of `%` from the SDK if its available